README.md

```diff
@@ -16,6 +16,10 @@ i.e. containing no JavaScript that runs on the client side.
 All API and CDN traffic is proxied through GoCook, there is no communication
 with any other hosts than GoCook itself.
 
+## Hosted instances
+
+- <https://cook.adminforge.de/>
```

Comment:
Great, thanks for your efforts! Then let's add the instance on lunar.icu too :tada: 

```suggestion
- <https://cook.adminforge.de/>
- <https://gocook.lunar.icu/>
```